google/cloud/bigtable/examples/bigtable_hello_world.cc

```diff
@@ -33,32 +33,32 @@ int main(int argc, char* argv[]) try {
   std::string const table_id = argv[3];
 
   // Connect to the Cloud Bigtable Admin API.
-  //! [connect admin] [START connecting_to_bigtable]
+  //! [connect admin] [START bigtable_hw_connect]
   google::cloud::bigtable::TableAdmin table_admin(
       google::cloud::bigtable::CreateDefaultAdminClient(
           project_id, google::cloud::bigtable::ClientOptions()),
       instance_id);
-  //! [connect admin] [END connecting_to_bigtable]
+  //! [connect admin] [END bigtable_hw_connect]
 
-  //! [create table] [START creating_a_table]
+  //! [create table] [START bigtable_hw_create_table]
   // Define the desired schema for the Table.
   auto gc_rule = google::cloud::bigtable::GcRule::MaxNumVersions(1);
   google::cloud::bigtable::TableConfig schema({{"family", gc_rule}}, {});
 
   // Create a table.
   auto returned_schema = table_admin.CreateTable(table_id, schema);
-  //! [create table] [END creating_a_table]
+  //! [create table]
```

Comment:
"END bigtable_hw_create_table" ?

Reply:
I intentionally made the scope of `creating_a_table` larger.  The examples in other languages (see following links) both create the Table (as in the "the thing inside the service that holds data") and the Table *handle* (as in "the thing in the application memory that gives you access to that thing in the service").

https://github.com/GoogleCloudPlatform/php-docs-samples/blob/5dfa97f1b8694cc22f4d633e1234ddb6865f56fe/bigtable/src/helloworld/create_table.php#L34-L80

https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/4a9929abc4c03ed97d7964dbecf1c9d48b1354d8/bigtable/hello_world.rb#L18-L30

https://github.com/googleapis/nodejs-bigtable/blob/0d8412140ceb32bf70837c6d18d5d2ce47ffba78/samples/hello-world/index.js#L38-L53

Reply:
I see.  Thanks.